gemfire-core/src/main/java/com/gemstone/gemfire/internal/LinuxProcFsStatistics.java

```diff
@@ -236,14 +236,14 @@ private static void getLoadAvg(double[] doubles, int[] ints) {
       if (threadLoad != null) {
         int slashIndex = threadLoad.indexOf('/');
         if (slashIndex >= 0) {
-          try {
-            ints[LinuxSystemStats.threadsINT] = Integer.parseInt(
-                threadLoad.substring(slashIndex + 1));
-          } catch (NumberFormatException ignored) {
-          }
+          threadLoad = threadLoad.substring(slashIndex + 1);
+        }
+        try {
+          longs[LinuxSystemStats.totalThreadsLONG] = Long.parseLong(threadLoad);
+        } catch (NumberFormatException ignored) {
```

Comment:
Should we also report the new processes showing up, which I guess is the last field. Just so that we would know whether other activities are happening on the host or is it dedicated for our system?